src/platforms/common/configuration/options.mdx

```diff
@@ -411,6 +411,18 @@ If the callback is not set, or it returns `undefined`, the default naming scheme
 
 </PlatformSection>
 
+<ConfigKey name="send-client-reports" supported={["java", "javascript", "dotnet", "python", "dart", "apple"]}>
```

Comment:
Hope I didn't miss any here. Do I explicitly need to include android? My testing showed the option without doing so.

Reply:
No need